working-together/README.md

```diff
@@ -4,156 +4,209 @@ A guide for working well together.
 
 ## Pair Programming
 
-**Before:**
+### Before:
 
-- Agree Scope (Pairing for two hours? Until the ticket is complete? Just get
-  past the bug?)
-- Agree Physical/Virtual Location ("Will we both be comfortable here?")
-- Agree Working Environment (Two keyboards? Text editor and other tools)
-- Agree Pairing Style (Time-based? [Ping pong]?)
+- **Agree Scope**
+  - Pairing for two hours?
+  - Until the ticket is complete?
+  - Just get past the bug?
 
-**During:**
+- **Agree Physical/Virtual Location**
+  - "Will we both be comfortable here?"
```

Comment:
For consistency with the other examples, should the quotation marks be removed here?
```suggestion
  - Will we both be comfortable here?
```

Reply:
Thanks ❤️